Helper/Config.php

```diff
@@ -103,4 +123,24 @@ public function getReleaseStage()
         return false;
     }
 
-}
\ No newline at end of file
+    /**
+     * Get the project_root full path from env.php if existent.
+     *
+     * This can also be the boolval `true` if it should be resolved using Magento.
+     *
+     * @return bool|mixed|string
+     */
+    public function getProjectRoot() {
+        if (array_key_exists('project_root', $this->bugsnagConfig)) {
+            $projectRoot = $this->bugsnagConfig['project_root'];
+            if ($projectRoot === true) {
+                // The root should be resolved by Magento
+                return $this->directoryList->getRoot();
+            } else {
```

Comment:
I thought about adding a `} if else (is_dir($projectroot)) {` check here, but figured this allows more flexibility? Thoughts?